hacks/values/kratos.yaml

```diff
@@ -23,6 +23,13 @@ kratos:
       - "--yes"
       - "--config"
       - "/etc/config/kratos.yaml"
+    resources:
```

Comment:
Sure we want to hardcode resources here?
Usually resources are not set by default in helm charts.

But i'm curretnly not sure what the files in hack do anyway?

Reply:
the hacks folder afaik contains sample values for all charts, which are used to perform `helm test`s. I checked the corresponding hydra values.yaml and also found resource set there.

Reply:
The values in hacks are to tests various parameters in the ci and verify if the charts are installable and don’t contain any semantic issues like invalid indentation (the bane of yamls…)